src/usePopper.js

```diff
@@ -98,35 +98,32 @@ export const usePopper = (
   ]);
 
   const popperInstanceRef = React.useRef();
-  const createPopper = React.useMemo(
-    () => options.createPopper || defaultCreatePopper,
-    [options.createPopper]
-  );
 
   useIsomorphicLayoutEffect(() => {
-    let popperInstance = null;
-    if (referenceElement != null && popperElement != null) {
-      popperInstance = createPopper(
-        referenceElement,
-        popperElement,
-        popperOptions
-      );
-
-      popperInstanceRef.current = popperInstance;
+    if (popperInstanceRef.current) {
+      popperInstanceRef.current.setOptions(popperOptions);
+    }
+  }, [popperOptions]);
```

Comment:
I've moved this `useIsomorphicLayoutEffect` before the other one.
I assume we only want to call `setOptions(...)` when the options have changed. Calling it immediately after creating the popper instance would be wasteful, right?